### ISSUE TITLE
Pkg: Separate form_item to its own macro

### ISSUE THREAD

**Issue by xak** [SAP/techne#1106]
See `form/component.njk`

There are instances where a `form_item` is needed without a group. This should include the form element and the label only.

```
    <div class="tn-form__item tn-form__item--inline tn-form__item--check">
        <input class="tn-form__control" type="checkbox" id="checkbox-1" name="checkbox-name-1" value="">
        <label class="tn-form__label" for="checkbox-1">List item 1</label>
    </div>
```

**Comment by xak** [comment]
This can be done with #960

**Comment by xak** [comment]
Implemented in #960 / pull request #1169
